Kernel/Arch/riscv64/boot.S

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023, Sönke Holz <sholz830@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+// In a specially-named text section so that the linker script can put it first in .text.
+.section ".text.first"
+
+.global start
+.type start, @function
+start:
+  // We expect that only one HART jumps here and that we are running in supervisor mode.
```

Comment:
Hart is not fully capitalized, as it’s a contraction of „hardware thread“, not an abbreviation. Cf. Wasm, not WASM.